src/qt/bitcoingui.h

```diff
@@ -19,6 +19,12 @@
 #include <QPushButton>
 #include <QSystemTrayIcon>
 
+#ifdef Q_OS_MAC
+#include <qt/macos_appnap.h>
+#endif
+
+#include <memory>
```

Comment:
is this addition needed? the bitcoin PR didn't add this

Reply:
Hmmm, true. Not sure how it got here 🙈 Dropped.